src/lineup/ALineUpView2.ts

```diff
@@ -78,6 +83,7 @@ export abstract class ALineUpView2 extends AView {
 
     this.$node.classed('lineup', true);
 
+    // hack in for providing the data provider within the graph
```

Comment:
It is all about the reference. Using this method, the reference to `.data` remain the same, while using `Promise.resolve(...)` the value would change